trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-facilities/facility-pricing-risk-table.component-test.js

```diff
@@ -1,4 +1,4 @@
-const componentRenderer = require('../../../../../../component-tests/componentRenderer');
+const componentRenderer = require('../../../../../componentRenderer');
 
 const component = '../templates/case/underwriting/pricing-and-risk/_macros/section-facilities/facility-pricing-risk-table.njk';
 
```

Comment:
Why has the folder been named 'pricing-and-risk1'?

Reply:
Renamed the folder to `pricing-and-risk`